packages/apollo-federation/src/composition/validate/sdl/__tests__/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
@abernix does this addition look sane? IIRC, we want a tsconfig at each `__tests__` folder level. Adding this resolved some TS errors in neighboring test files w.r.t. the use of `.flat()`

Reply:
Hmm.  The repeated `tsconfig.json` files in each `__tests__` folder is not ideal (I wasn't aware we were doing that!), but that should be going away in AS3's setup.

It would be great if we could avoid putting specific configuration in these dispersed files though and instead add the configuration to the `tsconfig.test.base` file and keep the individual extension files extremely minimal, e.g., like the one seen in [`composition/validate/__tests__`](https://github.com/apollographql/apollo-server/blob/bdf2217b713d1836426b221d0581834737153d7b/packages/apollo-federation/src/composition/validate/__tests__/tsconfig.json#L6).